tempesta_fw/http.h

```diff
@@ -453,10 +455,14 @@ typedef struct {
 	time_t			date;
 	time_t			last_modified;
 	unsigned long		jrxtstamp;
+	TfwHttpMsg		*req;
 } TfwHttpResp;
 
 #define TFW_HTTP_RESP_STR_START(r)	__MSG_STR_START(r)
 #define TFW_HTTP_RESP_STR_END(r)	((&(r)->body) + 1)
+#define TFW_HTTP_RESP_BOUND_REQ(r)	(TfwHttpReq *)(((TfwHttpResp *)r)->req)
```

Comment:
This appears to be a cut-n-paste bug. The macro is the same as `TFW_HTTP_REQ_BOUND_RESP()` above. One of these two should be different.

On another thought, I wouldn't make any of these a macro. The statement short enough, and it is totally clear so  there's no need to explain it through a different name (the name of a macro).